pkgs/development/python-modules/boinor/default.nix

```
@@ -0,0 +1,62 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  pythonAtLeast,
+  astropy,
+  astroquery,
+  jplephem,
+  matplotlib,
+  numba,
+  numpy,
+  pandas,
+  plotly,
+  pyerfa,
+  scipy,
+  flit-core,
+}:
+let
+  rev = "8e89b601b9d394a815562884c6fd39228d2b9f24";
+in
+buildPythonPackage rec {
```

it seems this `rec` is not used
```suggestion
buildPythonPackage {
```